json-parse.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+
+
+function jsonParse(data){
+  var reqData = JSON.parse(data.toString());
```

Comment:
What's the benefit of having this in a module instead of just calling ``` JSON.parse(data.toString()); ``` where it's needed?